core/src/main/java/bisq/core/dao/governance/proposal/ProposalService.java

```diff
@@ -219,12 +222,14 @@ private void publishToAppendOnlyDataStore() {
                 .filter(proposal -> validatorProvider.getValidator(proposal).isValidAndConfirmed(proposal))
                 .map(ProposalPayload::new)
                 .forEach(proposalPayload -> {
-                    boolean success = p2PService.addPersistableNetworkPayload(proposalPayload, true);
-                    if (success)
-                        log.info("We published a ProposalPayload to the P2P network as append-only data. proposalTxId={}",
-                                proposalPayload.getProposal().getTxId());
-                    else
-                        log.warn("publishToAppendOnlyDataStore failed for proposal " + proposalPayload.getProposal());
+                    UserThread.runAfterRandomDelay(() -> {
+                        boolean success = p2PService.addPersistableNetworkPayload(proposalPayload, true);
+                        if (success)
+                            log.info("We published a ProposalPayload to the P2P network as append-only data. proposalTxId={}",
+                                    proposalPayload.getProposal().getTxId());
+                        else
+                            log.warn("publishToAppendOnlyDataStore failed for proposal " + proposalPayload.getProposal());
+                    }, 100, 5000, TimeUnit.MILLISECONDS);
```

Comment:
```suggestion
                    }, 1, 1800, TimeUnit.SECONDS);
```

Reply:
We don't want to risk that the user shuts down the app in the meantime. The long we delay the higher the risk.

Reply:
There are a lot of clients and they will all be sending the same data more or less so it's not dependent on one single instance. If you think that's not as safe I'm ok with the 5s interval as well.

Reply:
Yes, maybe we need a more sophisticated solution here... or just increase the delay?

Reply:
Implemented as discussed. We set reBroadcast to false to avoid flooding. Other broadcast handling is less critical as user only publishes its own proposals/votes. But still some case where I added a longer random delay to avoid network stress.